CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 <!--## Unreleased-->
 
+## [2.0.0-alpha.38] - 2017-04-13
+
+* [minor breaking change] Revert of a change from alpha.37. The experimental _fork method on Analyzer returns an Analyzer again, not a Promise of an Analyzer.
```

Comment:
`_fork` kills the markdown italicization effect.  (on github it goes italics for days...)